proposals/serverlessworkflow.md

```diff
@@ -0,0 +1,170 @@
+
+Here is diagram that illustrates the Serverless Workflow functional flow:
+
+<p align="center">
+<img src="https://github.com/cncf/wg-serverless/raw/master/workflow/spec/media/sample-serverless-workflow2.png" with="400px" height="260px" alt="Serverless Workflow Diagram"/>
+</p>
+
+You can see the Serverless Workflow specification examples
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/spec-examples.md).
+
+You can see the Serverless Workflow specification use-cases
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/spec-usecases.md)
+
+You can see the Serverless Workflow specification logo
+[here](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/media/logo-small-text.png).
+
+The work on this specification is currently being done within the
+CNCF's Serverless WG Workflow sub-group, but with the large growth of the specification,
+and the short-term plan for a release of our first milestone (v0.1), it would make
+sense for this work to be more formalized as a new sandbox project under the CNCF.
+
+The goals/roadmap of the project include:
+
+- finalize release v0.1.
+- keep improving specification to v1.0.
+- define requirements for implementation compliance.
+- define requirements for Serverless Workflow API/SDK.
+
+**Statement on alignment with CNCF mission**:
+
+Being born out of the CNCF's Serverless Working Group, the Serverless Workflow
+project (and its members) share the CNCF's goals of promoting Cloud Native
+technologies, and offering choice to our consumers through an open
+interoperability specification, as shown by the significant participation
+from key industry companies.
+
+We believe that the CNCF provides the proper home for this due to its
+commitment to the promotion and development of open, vendor-neutral projects.
+Additionally, the wide breadth of the CNCF members will provide the feedback
+necessary to ensure the Serverless Workflow specification isn't too limited in its
+scope and appeals to as many constituents of the cloud native community
+as possible.
+
+**Sponsor / Advisor from TOC**:
+
+**Preferred maturity level**:
+
+Sandbox
+
+**License**:
+
+Apache License v2.0
+
+**Source control repositories**:
+
+CNCF Serverless WG repository (workflow directory): https://github.com/cncf/wg-serverless/tree/master/workflow
+
+**External dependencie**:
+
+None
+
+**Initial Maintainers**:
+
+We have the
+[GOVERNANCE](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/governance/readme.md)
+doc which explains the processes we follow.
+
+Also we have the:
+[OWNERS](https://github.com/cncf/wg-serverless/blob/master/workflow/spec/governance/owners.md)
+doc lists community members currently responsible for overseeing, reviewing, and approving changes to specification documents.
+
+**Infrastructure Requests**:
+
+We would like to request a dedicated github repository for the Serverless Workflow specification.
+
+We would like to request a dedicated website.
```

Comment:
you should probably get a dedicated mailing list too - if sandbox projects get those

Reply:
added. yes good point